nuxt/components/content/HomeExperience.vue

```diff
@@ -118,7 +118,7 @@
         >
           <NuxtImg
             v-for="url in ExperienceLogoWhiteSet"
-            class="max-h-10 w-full object-contain object-left"
+            class="max-h-10 w-[9rem] md:w-[11.5rem] object-contain object-left"
```

Comment:
I believe this is causing a problem with the viewport padding, right side. See screenshot 
<img width="382" alt="image" src="https://github.com/user-attachments/assets/d768b59e-081e-403a-85dc-b0f34fa7b85d">

Reply:
New headshot image scaling was the issue. Pushed a patch. Marking as resolved.